lib/__tests__/standalone-syntax.test.js

```diff
@@ -249,26 +249,33 @@ it('standalone with postcss-safe-parser', () => {
 
 		expect(results).toHaveLength(6);
 
+		const safeParserExtensionsTest = /\.(css|pcss|postcss)$/i;
```

Comment:
Maybe this should be a boolean since in both cases that's what we need?

Reply:
We need to test every result. It's possible to create new abstraction, but it will make code more complex and won't solve any problem.